turtle/src/main/kotlin/com/lordcodes/turtle/ShellScript.kt

```diff
@@ -94,9 +94,7 @@ class ShellScript constructor(workingDirectory: File? = null) {
         val exitCode = exitValue()
         if (exitCode != 0) {
             val errorText = errorStream.bufferedReader().use(BufferedReader::readText)
-            if (errorText.isNotEmpty()) {
-                throw ShellRunException(exitCode, errorText.trim())
-            }
+            throw ShellRunException(exitCode, errorText.trim())
```

Comment:
For me this if is a bug.
In bash `which xezrzueior || echo "tool not installed"` works but not in turtle without this change.
That's because `which` produces an error code but no error output

Reply:
Fair enough, I hadn't found any commands with no error output yet.